tests/test_orbit.py

```diff
@@ -4280,6 +4280,163 @@ def test_full_plotting():
+    assert numpy.isclose(o.pmra(), -3258.553), \
+        "PMRA of Lacaille 8760 does not match SIMBAD value"
+    assert numpy.isclose(o.pmdec(), -1145.396), \
+        "PMDec of Lacaille 8760 does not match SIMBAD value"
+    assert numpy.isclose(o.vlos(), 20.56), \
+        "radial velocity of Lacaille 8760 does not match SIMBAD value"
+
+    # test GJ 440 from SIMBAD
+    with pytest.raises(ValueError) as excinfo:
+        Orbit.from_name('GJ 440')
+    msg = "failed to find all necessary coordinates for GJ 440"
+    assert str(excinfo.value) == msg, \
+        "expected message '{}' but got '{}' instead".format(msg, str(excinfo.value))
+
+    # test Vega from Gaia
+    with pytest.warns(galpyWarning) as record:
+        warnings.filterwarnings('ignore', 'unclosed', ResourceWarning, 'socket')
+        o = Orbit.from_name('Vega', gaiadr2=True)
+    assert len(record) == 1, \
+        "Orbit.from_name('Vega', gaiadr2=True) threw {} warnings, but should have thrown 1".format(len(record))
+    msg = "failed to find Vega in Gaia; falling back on SIMBAD"
+    assert str(record[0].message) == msg, \
+        "expected warning message '{}' but got '{}' instead".format(msg, str(record[0].message))
+    assert numpy.isclose(o.ra(), 279.23473479), \
+        "RA of Vega did not default to SIMBAD when searching Gaia"
+    assert numpy.isclose(o.dec(), 38.78368896), \
+        "DEC of Vega did not default to SIMBAD when searching Gaia"
+    assert numpy.isclose(o.dist(), 1/130.23), \
+        "Parallax of Vega did not default to SIMBAD when searching Gaia"
+    assert numpy.isclose(o.pmra(), 200.94), \
+        "PMRA of Vega did not default to SIMBAD when searching Gaia"
+    assert numpy.isclose(o.pmdec(), 286.23), \
+        "PMDec of Vega did not default to SIMBAD when searching Gaia"
+    assert numpy.isclose(o.vlos(), -20.60), \
+        "radial velocity of Vega did not default to SIMBAD when searching Gaia"
+
+    # test Lacaille 8760 from Gaia
+    o = Orbit.from_name('Lacaille 8760', gaiadr2=True)
+    assert numpy.isclose(o.ra(), 319.29559940781115), \
+        "RA of Lacaille 8760 does not match Gaia value"
+    assert numpy.isclose(o.dec(), -38.87229724957863), \
+        "DEC of Lacaille 8760 does not match Gaia value"
+    assert numpy.isclose(o.dist(), 1/251.82949243686906), \
+        "Parallax of Lacaille 8760 does not match Gaia value"
+    assert numpy.isclose(o.pmra(), -3258.5531275249455), \
+        "PMRA of Lacaille 8760 does not match Gaia value"
+    assert numpy.isclose(o.pmdec(), -1145.3957291901002), \
+        "PMDec of Lacaille 8760 does not match Gaia value"
+    assert numpy.isclose(o.vlos(), 20.561701480854417), \
+        "radial velocity of Lacaille 8760 does not match Gaia value"
+
+    # test GJ 440 from Gaia
+    with pytest.raises(ValueError) as excinfo:
+        with pytest.warns(galpyWarning) as record:
+            warnings.filterwarnings('ignore', 'unclosed', ResourceWarning, 'socket')
```

Comment:
``ResourceWarning`` is not defined on python2, is this really necessary? If not, just remove it, if yes, then you need to test the python version...

Reply:
OK, I will make those changes.

I put in a filter for ResourceWarnings in the tests because socket throws a warning that the connection is unclosed. This is normally ignored by default in Python, but it is caught when I explicitly test for the warnings. I will remove the ResourceWarning part and replace it with a general filter for warnings from socket.